src/notation/view/pianokeyboard/pianokeyboardcontroller.cpp

```diff
@@ -34,6 +34,14 @@ void PianoKeyboardController::init()
     context()->currentNotationChanged().onNotify(this, [this]() {
         onNotationChanged();
     });
+
+    dispatcher()->reg(this, "piano-keyboard-toggle-notated-pitch", [this](const actions::ActionData& args) {
```

Comment:
It looks like you are registering both `PianoKeyboardController` and `PianoKeyboardPanelContextMenuModel` for this action. Having two handlers for one action is actually not ideal, and may stop working at some point. 

Instead, the controller should take this value from `INotationConfiguration::pianoKeyboardNotatedPitch`, and listen for changes via the channel from the returned `ValCh`.